From 40537a80edeba25853df745e969f1b0bd5bc71bf Mon Sep 17 00:00:00 2001
From: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
Date: Thu, 18 Dec 2014 16:13:58 +0200
Subject: [PATCH 1/1] raddb: Comment on ipaddr/ipv4addr/ipv6addr use

Describe combined ipaddr/ipv4addr/ipv6addr use in
raddb/sites-available/default.
---
 raddb/sites-available/default | 21 +++++++++++++--------
 1 file changed, 13 insertions(+), 8 deletions(-)

diff --git a/raddb/sites-available/default b/raddb/sites-available/default
index 77c271c..77602e4 100644
--- a/raddb/sites-available/default
+++ b/raddb/sites-available/default
@@ -84,17 +84,22 @@ listen {
 	#  source IP address for packets sent to a home server, the
 	#  proxy listeners are automatically created.
 
-	#  IP address on which to listen.
+	#  ipaddr/ipv4addr/ipv6addr - IP address on which to listen.
+	#  Out of several options the first one will be used.
+	#
 	#  Allowed values are:
-	#	dotted quad (1.2.3.4)
-	#       hostname    (radius.example.com)
-	#       wildcard    (*)
+	#	IPv4 address (e.g. 1.2.3.4, for ipv4addr/ipaddr)
+	#	IPv6 address (e.g. 2001:db8::1, for ipv6addr/ipaddr)
+	#	hostname     (radius.example.com,
+	#			A record for ipv4addr,
+	#       		AAAA record for ipv6addr,
+	#			A or AAAA record for ipaddr)
+	#       wildcard     (*)
+	#
+	# ipv4addr = *
+	# ipv6addr = *
 	ipaddr = *
 
-	#  OR, you can use an IPv6 address, but not both
-	#  at the same time.
-#	ipv6addr = ::	# any.  ::1 == localhost
-
 	#  Port on which to listen.
 	#  Allowed values are:
 	#	integer port number (1812)
-- 
2.1.3

